schemasheets/schemaview_vs_examples.py

```diff
@@ -0,0 +1,2 @@
+from linkml_runtime import SchemaView
+se_sv = SchemaView('examples/output/single_examples.yaml')
```

Comment:
this is an odd file - did you mean to check this in?

Reply:
yeah, I did. it should also be part of the missing text you noticed.